parser-typechecker/src/Unison/Server/Errors.hs

```diff
@@ -33,7 +33,7 @@ badHQN hqn = err400
 backendError :: Backend.BackendError -> ServerError
 backendError = \case
   Backend.NoSuchNamespace n ->
-    noSuchNamespace . Path.toText $ Path.unabsolute n
+    noSuchNamespace . Path.toText $ n
```

Comment:
Same here, this adds a dot prefix which would otherwise not be there, but seems like it probably should be.